pkg/reconciler/configuration/config/testdata/config-defaults.yaml

```diff
@@ -0,0 +1,122 @@
+# Copyright 2020 The Knative Authors
```

Comment:
This should a link to the `./config/core/configmaps/defaults.yaml` rather than just a file.

Reply:
so [ConfigMapFromTestFile](https://github.com/knative/pkg/blob/f0f096d812928f49f0cc1b52437bcd27cfff8334/configmap/testing/configmap.go#L45) is not implemented to follow symlinks should I add that?

Reply:
All our other store/cm tests load symlinks without much trouble. It'll work here as well, I think?

Reply:
done